server.js

```diff
@@ -42,6 +42,14 @@ const prerender = (req, res, store) => {
   const user = {
     key: process.env.EXAMPLE_USER_KEY
   };
+
+  if(process.env.EXAMPLE_USER_CUSTOM_JSON) {
+    try {
+      user.custom = JSON.parse(process.env.EXAMPLE_USER_CUSTOM_JSON);
+    } catch(e) {
+      console.log("Unable to parse env variable EXAMPLE_USER_CUSTOM_JSON: ", process.env.EXAMPLE_USER_CUSTOM_JSON, e);
```

Comment:
I believe each param sent over to `console.log` will be separated by a space, so the trailing space in the first param could be removed

Reply:
√ fixed!